platforms/jvm/testing-jvm/src/test/groovy/org/gradle/api/tasks/testing/TestTaskSpec.groovy

```diff
@@ -199,7 +199,7 @@ class TestTaskSpec extends AbstractProjectBuilderSpec {
         def closure = Mock(Closure)
 
         given:
-        expectTestSuitePasses()
```

Comment:
Why did these change?

Reply:
It's because `expectTestSuitePasses()` only sends an event for the top-level suite (i.e. parent = null), which does not increment the test count at all, so it appears like no tests were discovered and we get the failure.  These tests don't have anything to do with whether tests execute or not, so it doesn't impact them if we change it to `expectTestPasses()` instead.